### ISSUE TITLE
Cannot enter OTP before login

### ISSUE THREAD

**Issue by FlorianPfaff** [microsoft/vscode-remote-release#5824]
Hi,

the SSH plugin is definitely great! However, maybe my use case has not come up yet in its tests: I am trying to log in on a server that requires OTPs (one time passwords) to be entered *before* one can enter the actual user password. Sometimes, I can enter it in the "terminal" tab - and sometimes, the prompt only shows up in my log and I cannot type in anything. This is shown in the screenshot here:
![image](https://user-images.githubusercontent.com/6773539/140419416-02b9efae-52e4-41ac-b31e-61998a75855f.png)
On Powershell or WinSCP, the prompt always comes up, and I can type in the OTP, so I think this is not a server-side problem.

Thanks for considering solving this issue!

Florian

**Comment by tanhakabir** [comment]
Hey! Have you tried have `remote.SSH.showLoginTerminal` enabled?

![image](https://user-images.githubusercontent.com/12758612/140422473-70bb6367-ad9e-42d8-87d7-dbb219c92f3f.png)

**Comment by FlorianPfaff** [comment]
Thank you, this resolved the issue. In case any changes are considered to improve the support here and making it supported without this checkbox, I think I discovered some helpful pattern before I read your message. Without this checkbox, one obviously needs to click on "details" while connecting to see the terminal tab. The problem that I cannot type in anything seemed to _not_ occur when I waited for a while before clicking on "details". So maybe this issue only occurs if the prompt comes up while the "details" window is currently popping up. But no guarantee on this - your fix definitely works, thanks again!
